docs/cheatsheet/insert.rst

```diff
@@ -97,29 +97,104 @@ Insert several nested objects at once:
 ----------
 
 
-"Upserts" as well as and other combinations of :eql:stmt:`INSERT` and
-some alternative operation are possible:
+Sometimes it's necessary to check whether some object exists and
+create it if it doesn't. If this type of object has an exclusive
+property, the ``UNLESS CONFLICT`` clause can make the ``INSERT``
+command indempotent. So running such a command would guarantee that a
+copy of the object exists without the need for more complex logic:
+
+.. code-block:: edgeql
+
+    # Try to create a new User
+    INSERT User {
+        name := "Łukasz Langa",
```

Comment:
I'm sure Łukasz wouldn't mind, but I think it'll be better to use less identifying names in examples.

Reply:
I can use something more generic, but I think we already have `Łukasz` in a couple of places (probably because examples used were matching stuff used in blog posts).

Reply:
Let's use something generic.